provider/marathon.go

```diff
@@ -21,13 +21,14 @@ import (
 
 // Marathon holds configuration of the Marathon provider.
 type Marathon struct {
-	BaseProvider     `mapstructure:",squash" description:"go through"`
-	Endpoint         string `description:"Marathon server endpoint. You can also specify multiple endpoint for Marathon"`
-	Domain           string `description:"Default domain used"`
-	ExposedByDefault bool   `description:"Expose Marathon apps by default"`
-	Basic            *MarathonBasic
-	TLS              *tls.Config
-	marathonClient   marathon.Marathon
+	BaseProvider       `mapstructure:",squash" description:"go through"`
```

Comment:
The `StructTag` is useless ...
I think I forgot to remove this one...
@emilevauge : Can you delete it ?